WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -115,24 +117,35 @@ public SendViewModel(Wallet wallet, IObservable<Unit> balanceChanged, Observable
 					return allFilled && !hasError;
 				});
 
-		NextCommand = ReactiveCommand.CreateFromTask(async () =>
-		{
-			var address = BitcoinAddress.Create(To, wallet.Network);
+		NextCommand = ReactiveCommand.CreateFromTask(
+			async () =>
+			{
+				var address = BitcoinAddress.Create(To, _wallet.Network);
 
-			_transactionInfo.Reset();
-			_transactionInfo.Amount = new Money(AmountBtc, MoneyUnit.BTC);
+				_transactionInfo.Reset();
+				_transactionInfo.Amount = new Money(AmountBtc, MoneyUnit.BTC);
+				_transactionInfo.IsAutomaticSelectionEnabled = IsAutomaticSelectionEnabled;
 
-			var labelDialog = new LabelEntryDialogViewModel(_wallet, _transactionInfo);
-			var result = await NavigateDialogAsync(labelDialog, NavigationTarget.CompactDialogScreen);
-			if (result.Result is not { } label)
-			{
-				return;
-			}
+				var labelDialog = new LabelEntryDialogViewModel(_wallet, _transactionInfo);
+				var result = await NavigateDialogAsync(labelDialog, NavigationTarget.CompactDialogScreen);
+				if (result.Result is not { } label)
+				{
+					return;
+				}
 
-			_transactionInfo.UserLabels = label;
+				_transactionInfo.UserLabels = label;
 
-			Navigate().To(new TransactionPreviewViewModel(wallet, _transactionInfo, address, _isFixedAmount));
-		}, nextCommandCanExecute);
```

Comment:
Seems like this code is large enough to place it in its own method, no? `async Task OnNext()` 
CodeScene seems to be complaining about "method too large".